ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -310,6 +312,40 @@ public void setSelectionColor(ReactEditText view, @Nullable Integer color) {
     } else {
       view.setHighlightColor(color);
     }
+
+    setCursorColor(view, color);
+  }
+
+  private void setCursorColor(ReactEditText view, @Nullable Integer color) {
+    // Evil method that uses reflection because there is no public API to changes
+    // the cursor color programatically.
+    // Based on http://stackoverflow.com/questions/25996032/how-to-change-programatically-edittext-cursor-color-in-android.
+    try {
+      // Get the cursor resource id.
+      Field field = TextView.class.getDeclaredField("mCursorDrawableRes");
+      field.setAccessible(true);
+      int drawableResId = field.getInt(view);
+
+      // Get the editor.
+      field = TextView.class.getDeclaredField("mEditor");
+      field.setAccessible(true);
+      Object editor = field.get(view);
+
+      // Get the drawable and set a color filter.
+      Drawable drawable = view.getContext().getDrawable(drawableResId);
+      if (color != null) {
+        drawable.setColorFilter(color, PorterDuff.Mode.SRC_IN);
+      }
+      Drawable[] drawables = {drawable, drawable};
+
+      // Set the drawables.
+      field = editor.getClass().getDeclaredField("mCursorDrawable");
+      field.setAccessible(true);
+      field.set(editor, drawables);
+    } catch (Exception ex) {
+      // Ignore errors to avoid crashing if these private fields don't exist on modified
```

Comment:
Wondering if we should log this, what do you think? Seems like the kind of thing people would also want to have insight into ex: how many users are getting a slightly different UI because their Android doesn't support these fields?